docs/contributing-tutorial.md

```diff
@@ -349,6 +346,44 @@ This will add a command which prints `Hello OSTree!` when `ostree hello-ostree`
         $ ostree hello-ostree
         Hello OSTree!
 
+### Add new function to libostree
+
+This will add a function `ostree_kernel_args_testing()` in `src/libostree/ostree-kernel-args.c`.
+
+1. Add the following to `src/libostree/ostree-kernel-args.h`:
+        _OSTREE_PUBLIC
+        void ostree_kernel_args_testing (const char *arg); 
```

Comment:
I'd prefer something showing an example like 
```
gboolean ostree_kernel_args_foo (const char *arg, GCancellable *cancellable, GError **error)
```
which is a more common form for public API functions.